ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -273,51 +333,57 @@ Control {
                     image.fillMode: Image.PreserveAspectCrop
                     image.source: model.imageUrl ?? ""
 
-                    RowLayout {
+                    Control {
+                        width: (amountText.contentWidth + Style.current.padding)
+                        height: 24
                         anchors.left: parent.left
-                        anchors.leftMargin: Style.current.halfPadding
-                        anchors.bottom: parent.bottom
-                        anchors.bottomMargin: Style.current.halfPadding
-                        anchors.right: parent.right
-                        anchors.rightMargin: Style.current.halfPadding
-
-                        Control {
-                            Layout.maximumWidth: parent.width
-                            horizontalPadding: Style.current.halfPadding
-                            verticalPadding: Style.current.halfPadding/2
-                            visible: !!model.id
-
-                            background: Rectangle {
-                                radius: Style.current.halfPadding/2
-                                color: Theme.palette.indirectColor2
-                            }
-                            contentItem: StatusBaseText {
-                                font.pixelSize: 13
-                                font.weight: Font.Medium
-                                maximumLineCount: 1
-                                elide: Text.ElideRight
-                                text: `#${model.id}`
-                            }
+                        anchors.leftMargin: 12
+                        anchors.top: parent.top
+                        anchors.topMargin: 12
+                        visible: (model.userHas > 1)
```

Comment:
What is this `userHas`? :thinking:

Reply:
TBD, we need to show the number if the user has more than 1 of each collectible ([figma](https://www.figma.com/file/ibJOTPlNtIxESwS96vJb06/%F0%9F%91%A4-Profile-%7C-Desktop?type=design&node-id=2460-31491&mode=dev)). not sure how to name the role